components/ListSurfspotsItems.js

```diff
@@ -1,19 +1,24 @@
 import styled from "styled-components";
 import Link from "next/link";
+import Image from "next/image";
+import arrow from "../public/images/arrow.png";
 
 styled;
 export default function ListSurfspotsItems({ name, link }) {
   return (
     <StyledLi>
-      <StyledLink href={link}>{name}</StyledLink>
+      <StyledLink href={link}>
+        {name} <StyledImage src={arrow} alt="arrow" width={30} height={30} />
+      </StyledLink>
     </StyledLi>
   );
 }
 
 const StyledLink = styled(Link)`
   text-decoration: none;
   color: #343436;
-  font-size: 20px;
+  font-size: 30px;
+  width: 100%;
 `;
 
 const StyledLi = styled.li`
```

Comment:
you can shrink down the margin to: 
margin: 10px 0px
the first value is for top and bottom, and the second for right and left.

Reply:
True :)